src/main/kotlin/ca/kittle/capabilities/party/models/Party.kt

```diff
@@ -1,13 +1,22 @@
 package ca.kittle.capabilities.party.models
 
-import kotlinx.serialization.Contextual
+import ca.kittle.capabilities.pc.models.Name
+import ca.kittle.util.randomUUIDv7
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
-import java.util.UUID
 
 @Serializable
 data class Party(
-    val name: String,
-    @Contextual @SerialName("_id")
-    val id: String = UUID.randomUUID().toString(),
+    val name: Name,
+    val partyMembers: List<PartyMember> = listOf(),
+    @SerialName("_id")
+    val id: PartyId = PartyId(),
 )
+
+object NewId {
+    operator fun invoke() = randomUUIDv7().toString()
+}
+
+@JvmInline
```

Comment:
**suggestion (performance):** Evaluate performance impact of value classes

While value classes provide type safety, they might introduce boxing/unboxing operations. Profile the application to ensure this doesn't negatively impact performance, especially for frequently used types like PartyId.